src/pyvcam_driver.py

```diff
@@ -0,0 +1,537 @@
+
+        :param num_frames: Number of frames to capture in the sequence.
+        :param exp_time: The exposure time.
+        :param interval: The time in milliseconds to wait between captures.
+
+        :return: A 3D np.array containing the pixel data from the captured frames.
+        """
+        return self.cam.get_sequence(num_frames, exp_time, timeout_ms, interval)
+    
+    def start_live(self, exp_time: int=None, buffer_frame_count: int=16,
+                   stream_to_disk_path: str=None) -> None:
+        """
+        Calls the pvc.start_live function to setup a circular buffer acquisition.
+
+        :param exp_time: The exposure time.
+        :param buffer_frame_count: The number of frames in the circular frame buffer.
+        :param stream_to_disk_path: The file path for data written directly to disk by PVCAM. 
+                                    The default is None which disables this feature.
+
+        :return: None.
+        """
+        self.cam.start_live(exp_time, buffer_frame_count, stream_to_disk_path)
+    
+    def start_seq(self, exp_time: int=None, num_frames: int=1) -> None:
+        """
+        Calls the pvc.start_seq function to setup a non-circular buffer acquisition.
+        This must be called before poll_frame.
+
+        :param exp_time: The exposure time.
+
+        :return: None.
+        """
+        self.cam.start_seq(exp_time, num_frames)
+    
+    def poll_frame(self, timeout_ms: int=WAIT_FOREVER, oldestFrame: bool=True, 
+                   copyData: bool=True) -> tuple[dict, float, int]:
+        """
+        Returns a single frame as a dictionary with optional meta data if available. 
+        This method must be called after either stat_live or start_seq and before either abort or finish. 
+        Pixel data can be accessed via the pixel_data key. Available meta data can be accessed via the meta_data key.
+
+        If multiple ROIs are set, pixel data will be a list of region pixel data of length number of ROIs. 
+        Meta data will also contain information for each ROI.
+
+        Use set_param(constants.PARAM_METADATA_ENABLED, True) to enable meta data.
+
+        :param timeout_ms: Duration to wait for new frames.
+        :param oldestFrame: If True, the returned frame will the oldest frame and will be popped off the queue. 
+                            If False, the returned frame will be the newest frame and will not be removed from the queue.
+        :param copyData: Selects whether to return a copy of the numpy frame which points to a new buffer, or the original numpy frame which points to the
+                         buffer used directly by PVCAM. Disabling this copy is not recommended for most situations. Refer to PyVCAM Wrapper.md for more details.
+
+        :return: A dictionary with the frame containing available meta data and 2D np.array pixel data, frames per second and frame count.
+        """
+        return self.cam.poll_frame(timeout_ms, oldestFrame, copyData)
```

Comment:
```suggestion
        return self.cam.poll_frame(timeout_ms, oldest_frame, copy_data)
```
Follow Python variable name convention.